core/chains/evm/txmgr/models.go

```diff
@@ -262,6 +267,16 @@ func (e EthTx) GetLogger(lgr logger.Logger) logger.Logger {
 		if meta.FwdrDestAddress != nil {
 			lgr = lgr.With("FwdrDestAddress", *meta.FwdrDestAddress)
 		}
+
+		if len(meta.MessageIDs) > 0 {
+			for _, mid := range meta.MessageIDs {
+				lgr = lgr.With("messageID", mid)
+			}
+		}
+
+		if len(meta.SeqNumbers) > 1 {
+			lgr = lgr.With("SeqNumbersMin", meta.SeqNumbers[0], "SeqNumbersMax", meta.SeqNumbers[1])
```

Comment:
Why does this assume SeqNumbers just has 2 elements, and sorted?
Is this is true, then why not break into 2 members, SeqNumbersMin and SeqNumbersMax?